lsh/indexing/HashTable.java

```diff
@@ -49,7 +49,7 @@ public HashBucket getHashBucket(SearchableObject searchableObject) {
 		
 		List<Double> objectFeatures = searchableObject.getObjectFeatures();
 		
-		List<Integer> objectHashBucket = new ArrayList<Integer>(this.numberOfHashFunctions);
```

Comment:
confused at why the removal of the instantiation here? i think this removal will lead to a deficiency in terms of speed due to resizing